node/coinstacks/polygon/daemon/init.sh

```diff
@@ -13,20 +13,23 @@ function extract_files() {
       continue
     fi
     if echo "$line" | grep -q "bulk"; then
-      wget -nc --timeout 0 --retry-connrefused --tries 0 $line -O - | zstd -cd | tar -xf - -C $CHAINDATA_DIR
+      wget -nc --timeout 0 --retry-connrefused --tries 0 $line -O - | zstd -cd | tar -xvkf - -C $CHAINDATA_DIR
     else
-      wget -nc --timeout 0 --retry-connrefused --tries 0 $line -O - | zstd -cd | tar -xf - -C $CHAINDATA_DIR --strip-components=3
```

Comment:
Just a remark - if our startup logic continues to grow perhaps we should consider using a more expressive language than bash for the init logic